views/includes/documents/Terms-of-Service.md

```diff
@@ -38,15 +38,15 @@ Within moderation we understand the occasional necessity to conserve device stor
+* User Scripts have an implied license of [MIT License (Expat)](https://opensource.org/licenses/MIT) if the `@license` *(including the alternate spelling of `@licence`)* metadata block key is absent.
+* Library Scripts need to include one reference to their [acceptable license(s)](#acceptable-licensing) either via a License Header *([sample MPL header](https://www.mozilla.org/MPL/headers/) if that license is chosen)*, a cross-site compatible metadata block with at least one `@license` or `@licence` key **and** one `@exclude *`, or full License text *(full text is not recommended for storage constraints on portable devices)*. Libraries also have an implied license of [MIT License (Expat)](https://opensource.org/licenses/MIT) in the absence of any licensing *(however this should not happen)*. It may also be useful to indicate in the source that it is a library and not a userscript. If the type of script is unclear the library may be subject to removal.
 
 #### Acceptable Licensing
-* Licensing must be [Open Source Initiative *(OSI)* approved](http://opensource.org/licenses/category) and declared with the name of the License in the Source Code for all published works. e.g. `@license` or `@licence` keys, License Header or full License text *(full text is not recommended for storage constraints on portable devices)*
+* Licensing must be [Open Source Initiative *(OSI)* approved](https://opensource.org/licenses/category) and declared with the name of the License in the Source Code for all published works. e.g. `@license` or `@licence` keys, License Header or full License text *(full text is not recommended for storage constraints on portable devices)*
 * If a Content only license is included then the [Default Code Licensing](#default-code-licensing) will be applied with licensing arbitration, but may not always be visibly indicated on a script home page.
 * Any license that prohibits derivatives is eligible for removal.
 * Non-reusable licenses, of course, may not be used.
-* [Public domain type](http://opensource.org/faq#public-domain) licenses may mean different things in different jurisdictions and therefore are not allowed and eligible for removal.
+* [Public domain type](https://opensource.org/faq#public-domain) licenses may mean different things in different jurisdictions and therefore are not allowed and eligible for removal.
 * Any attempt to unlicense a published work will be eligible for removal.
 * In the event of any code licensing dispute resolution the [Default Code Licensing](#default-code-licensing) shall prevail if previously undeclared and/or undocumented.
 
```

Comment:
> yet I did not want someone of the team members to wonder why they need to do despairing repetitive actions.

That's why issues should be made first... then discuss... etc... as [I described at the PR itself first](https://github.com/OpenUserJs/OpenUserJS.org/pull/913#issuecomment-189735612). That way you won't be doing repetitive actions e.g. unnecessary work. You did find one enhancement for wikipedia so that is something you could do with a probably +1 support for merging.

Sidebar: With wikipedia links always watch that the link is language neutral too. Typically here in the US it will redirect to https://en.wikipedia.org and we shouldn't be linking to en only pages if they have their own browser language detection when possible. Same goes for Mozilla links and any others that apply.